docs/dev-guide/javalang/foundations.md

```diff
@@ -6,6 +6,13 @@ sidebar_position: 1
 description: The Foundations section of the Temporal Developer's guide covers the minimum set of concepts and implementation details needed to build and run a Temporal Application – that is, all the relevant steps to start a Workflow Execution that executes an Activity.
 slug: /dev-guide/java/foundations
 toc_max_heading_level: 4
+tags:
+- guide-context
+- developer-guide
+- Java
+- java
```

Comment:
Need both casings here?

Reply:
weird.

Reply:
actually that is because this is generated -
we should make sure all are lowercase.
but this isn't a big blocked for the moment.